layout/_macro/post.swig

```diff
@@ -249,7 +249,7 @@
     {% endif %}
 
 
-    <div class="post-body" itemprop="articleBody">
+    <div class="post-body han-init-context" itemprop="articleBody">
```

Comment:
But it's will be in default class style.
<div class="post-body{% if theme.han %} han-init-context{% endif %}" itemprop="articleBody">